docs/user/tutorial/index.rst

```diff
@@ -95,16 +87,11 @@ Importing the project to Read the Docs
 
 To import your GitHub project to Read the Docs:
 
-#. Click the :guilabel:`Import a Project` button on your `dashboard <https://readthedocs.org/dashboard/>`_.
+#. Click the :guilabel:`Add project` button on your `dashboard <https://readthedocs.org/dashboard/>`_.
 
-#. Click the |:heavy_plus_sign:| button to the right of your ``rtd-tutorial`` project. If the list of repositories is empty, click the |:arrows_counterclockwise:| button.
+#. Click on :guilabel:`Install GitHub App on repository`, and choose your account and select the repository you created in the previous step.
 
-   .. figure:: /_static/images/tutorial/rtd-import-projects.gif
```

Comment:
Just to note that we are deleting an image here. I know that @agjohnson wanted to re-new/re-take them using the new dashboard.